src/common/lib/crypto.js

```diff
@@ -1,10 +1,10 @@
 import { AES, enc } from "crypto-js";
 
 export function encryptData(data, password, salt) {
-  return AES.encrypt(JSON.stringify(data), password + salt).toString();
+  return AES.encrypt(JSON.stringify(data), password).toString();
```

Comment:
No salt?

Reply:
it is too complicated to calculate a salt per installation for now. We just require a strong password.